Automattic/wp-calypso
# Photos Picker: Update migration connection notification

Closes https://github.com/Automattic/wp-calypso/issues/96006

## Proposed Changes

* Rename obsolete Picasa migration file (the logic remain the same)
* Update migration connection component from "Google Photos" to "Google Photos Picker" API

## Why are these changes being made?


* The changes have been made to cover the case when the previous Google Photos connection already exists, and we need to force the reconnection

## Testing Instructions



* Go to `/marketing/connections/{SITE_SLUG}`
* Make sure that you have established a connection with your Google Photos account
* Setup your sandbox and apply DIFF: D164673-code
  * Update `should_force_google_photos_picker` method response to `true` on the file: `google-photos.php`
* Refresh the same page and check if there is a notification for the service reconnection (check the screenshot)

<img width="1665" alt="Screenshot 2024-11-01 at 23 51 18" src="https://github.com/user-attachments/assets/3a23988e-6c51-4c00-9269-eb140f519422">

## Pre-merge Checklist
